drivers/auxdisplay/auxdisplay_serlcd.c

```diff
@@ -84,6 +74,8 @@ struct auxdisplay_serlcd_data {
 	bool blinking;
 	uint16_t cursor_x;
 	uint16_t cursor_y;
+	int command_delay_ms;
+	int special_command_delay_ms;
 };
```

Comment:
These should be uint, not int, and should be part of the config, not the data

Reply:
Moved to config.

And changed to uint as requested